src/components/Menu.jsx

```diff
@@ -1,11 +1,29 @@
-import React from 'react'
+import React, { useState, useEffect } from "react";
+import { Segment, List } from "semantic-ui-react";
+import MenuItem from "./MenuItem";
+import axios from "axios";
 
 const Menu = () => {
+  const [menuItems, setIdentification] = useState([]);
+  useEffect(() => {
+    axios.get("https://slowfood.heroku.com/api/products").then((response) => {
```

Comment:
Hmm... Is `https://slowfood.heroku.com` the real URL? Also, you are intercepting a call to `localhost:3000` in your tests.

Reply:
That is not a real url, just a placeholder. localhost:3000 was removed from the test in the last commit

Reply:
Okay